src/GUniLPOracle.t.sol

```diff
@@ -570,10 +570,16 @@ contract GUniLPOracleTest is DSTest {
         assertGt(uint256(liquidityEnd), uint256(liquidityOrig));
     }
 
-    function test_sqrt_price_overflow_fuzz(uint256 p0, uint256 dec0, uint256 p1, uint256 dec1) public {
-        uint256 MAX_PRICE = 1e12 * WAD;     // Max underlying asset Oracle price supported is $1 Trillion USD
-        uint256 MAX_DEC = 18;
+    // --- Fuzz ---
+    uint256 MAX_PRICE = 1e12 * WAD;   // Max underlying asset Oracle price supported is $1 Trillion USD
+    uint256 MIN_PRICE = 10**16;       // $0.01 USD
```

Comment:
Was this deducted by trial and error? I assume if we use smaller MAX_PRICE then MIN_PRICE can also be smaller.

Reply:
The `MIN_PRICE` was applied to have a realist price range for the general case.
Open to change it in favour of a more pragmatic range.

Reply:
There are definitely tokens with unit prices lower than $0.01 so this should be the theoretical lower limit from Tal's assessment.

Reply:
Maybe try to enforce the lower limit of one token according to the randomized other token. You can try to use  these outcomes from the assesment:
10^10 >> token1_usd_price / token0_usd_price
token0_usd_price / token1_usd_price < 10^19